### ISSUE TITLE
implementers

### ISSUE THREAD

**Issue by sparrell** [oasis-tcs/openc2-oc2ls#182]
This is a child of #143

```
Page 23, Line 12: suggest adding "implementers" before "MUST NOT" for clarity.

Page 23, Line 24: suggest adding "implementers" before "MUST NOT" for clarity.

```

**Comment by jmbrule** [comment]
"implementers MUST not use this in actual implementations"  seems awk.  Will you consider
"implementers MUST not use this in actual operations" or something else?

**Comment by romanojd** [comment]
I've addressed this issue within issue #143. Closing this comment.
